src/o365/spo/commands/listitem/listitem-batch-add.ts

```diff
@@ -52,110 +56,170 @@ class SpoListItemAddCommand extends SpoCommand {
     telemetryProps.folder = typeof args.options.folder !== 'undefined';
     return telemetryProps;
   }
-
-  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
-    let lineNumber: number = 0;
-    let contentTypeName: string | null = null;
-    let listRestUrl: string | null = null;
-    let maxBytesInBatch: number = 1000000; // max is  1048576
-    let rowsInBatch: number = 0;
-    let batchCounter = 0;
-    let recordsToAdd = "";
-    let csvHeaders: Array<string>;
-    const sendABatch = (batchCounter: number, rowsInBatch: number, changeSetId: string, recordsToAdd: string): Promise<any> => {
-      
-      let batchContents = new Array();
-      let batchId = v4();
-      batchContents.push('--batch_' + batchId);
-      cmd.log(`Sending batch #${batchCounter} with ${rowsInBatch} items`);
-      batchContents.push('Content-Type: multipart/mixed; boundary="changeset_' + changeSetId + '"');
-      batchContents.push('Content-Length: ' + recordsToAdd.length);
-      batchContents.push('Content-Transfer-Encoding: binary');
-      batchContents.push('');
-      batchContents.push(recordsToAdd);
-      
-      batchContents.push('--batch_' + batchId+'--');
-      cmd.log(batchContents);
-      const updateOptions: any = {
-        url: `${args.options.webUrl}/_api/$batch`,
-        headers: {
-          'Content-Type': `multipart/mixed; boundary="batch_${batchId}"`
-        },
-        body: batchContents.join('\r\n')
-      };
-      return request.post(updateOptions);
-    }
-    const parseResults = (response: any, cmd: CommandInstance, cb: (err?: any) => void): void => {
-      let responseLines: Array<string> = response.toString().split('\n');
-      // read each line until you find JSON... 
-      for (let responseLine of responseLines) {
-        try {
-          //check for error 
-
-          if (responseLine.startsWith("HTTP/1.1 5")) { //any 500 errors (like timeout), just stop
-            cmd.log("An HTTP 5xx error was returned from SharePoint. Please retry with a lower --batchsize ")
-            cb(responseLine);
-          }
-          // parse the JSON response...
-          var tryParseJson = JSON.parse(responseLine);
-          for (let result of tryParseJson.d.AddValidateUpdateItemUsingPath.results) {
-            if (result.HasException) {
-              cmd.log(result)
-            }
+  public static parseResults(response: string, cmd: CommandInstance, cb: (err?: any) => void): void {
+    cmd.log(typeof response)
+    let responseLines: Array<string> = response.toString().split('\n');
```

Comment:
You may have forgot to check for the verbose flag before some cmd.log calls in this file.

Reply:
I removed that log. was just for testing